lib/client/client_store.go

```diff
@@ -230,7 +230,8 @@ func (s *Store) FullProfileStatus() (*ProfileStatus, []*ProfileStatus, error) {
 		}
 		status, err := s.ReadProfileStatus(profileName)
 		if err != nil {
-			return nil, nil, trace.Wrap(err)
+			s.log.WithError(err).Debugf("skipping profile %q due to error", profileName)
```

Comment:
should this be a warning? A corrupted profile is a problem and the user might not understand why they cannot log in.

Reply:
If they cannot log in, then they should get a real error when they try to log in.

No strong preference from my end whether this is a warning or debug. Let me know what you prefer.